src/main/resources/application.properties

```diff
@@ -1,15 +1,26 @@
 # Zeebe endpoint
 ######################
 # in cloud:
-#zeebe.client.cloud.region=bru-2
-#zeebe.client.cloud.clusterId=XXX
-#zeebe.client.cloud.clientId=XXX
-#zeebe.client.cloud.clientSecret=XXX
+#camunda.client.mode=saas
+#camunda.client.auth.client-id=XXX
+#camunda.client.auth.client-secret=XXX
+#camunda.client.cluster-id=XXX
+#camunda.client.region=XXX
 
 # or local
-#zeebe.client.broker.gateway-address=127.0.0.1:26500
-#zeebe.client.security.plaintext=true
+#camunda.client.mode=simple
 
+# or selfmanaged
+camunda.client.mode=selfmanaged
+camunda.client.zeebe.base-url=http://localhost:26500
+camunda.client.auth.issuer=http://localhost:8080/auth/realms/camunda-platform/protocol/openid-connect/token
+camunda.client.zeebe.grpc-address=https://dave02.gke.c8sm.com
+camunda.client.zeebe.rest-address=https://dave02.gke.c8sm.com/rest
+camunda.client.zeebe.prefer-rest-over-grpc=false
+
+camunda.client.auth.client-id=zeebe
+camunda.client.auth.client-secret=KLn2WqAIbs
+camunda.client.identity.base-url=https://dave02.gke.c8sm.com/identity
```

Comment:
Great example but it should probably be commented out. The resulting Docker image should not be pre-configured for a special environment other than maybe localhost with default port and no TLS.